org.eclipse.jdt.core/model/org/eclipse/jdt/internal/core/CompilationUnit.java

```diff
@@ -247,8 +247,6 @@ protected boolean buildStructure(OpenableElementInfo info, final IProgressMonito
 		parser.setMethodsFullRecovery(true);
 		parser.setStatementsRecovery((reconcileFlags & ICompilationUnit.ENABLE_STATEMENTS_RECOVERY) != 0);
 
-		if (!computeProblems && !resolveBindings && !createAST) // disable javadoc parsing if not computing problems, not resolving and not creating ast
```

Comment:
The JavaDoc information is required during compilation; hence, it is being enabled.